ui-tests/src/test/resources/features/customizations/jdbc-driver-extension.feature

```diff
@@ -0,0 +1,16 @@
+@jbdc-driver-import
```

Comment:
please, add @wip tag as this is not yet complete oracle jdbc integration test. As now, we have agreed on the process of Oracle JDBC driver import, I would expect it not just to create the database connection, but actually insert something/read from the oracle DB